recipe/gcp-version.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/exporter/opentelemetry-exporter-jaeger-proto-grpc/setup.cfg b/exporter/opentelemetry-exporter-jaeger-proto-grpc/setup.cfg
+index 36a5d2b27..2bc9200fc 100644
```

Comment:
https://github.com/open-telemetry/opentelemetry-python/issues/2778

I can't reproduce the above issue even though this recipe is now running the full pytest suite on each of the subpackages. I suspect therefore that it is a thorny issue with binary dependencies that conda just happens to do better on. For this reason I'm relaxing this constraint to make it consistent with the rest of the packages, which will in turn allow us to restore the linux-ppc64le builds